index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <nav class="navbar flex flex-jc-sb flex-ai-c">
+        <div class="navbar--title">
+            <h1>Portfolio</h1>
+        </div>
+        <div class="navbar--links">
+            <ul class="flex flex-ai-c">
+                <li><a href="#">About</a></li>
+                <li><a href="">Work</a></li>
+                <li><a href="">Contact</a></li>
+            </ul>
+        </div>
+    </nav>
+    <section class="home">
+        <h1>Portfolio page for Evans and Abdulbasit</h1>
+        <p>Professional web developers</p>
+    </section>
+    <section class="work">
+        <div class="card">
+            <img src="./img/Cal.JPEG" />
+            <div class="text"><span>
+                    < </span>Calculator <span> /> </span></div>
+        </div>
+        <div class="card">
+            <img src="./img/Calc1.JPEG" alt="">
+            <div class="text"><span>
+                    < </span>Calculator <span> /> </span></div>
+        </div>
+        <div class="card">
+            <img src="./img/coding.JPEG" alt="">
+            <div class="text"><span>
+                    < </span>Ecommerce <span> /> </span></div>
+        </div>
+        <div class="card">
+            <img src="./img/comp.JPEG" alt="">
+            <div class="text"><span>
+                    < </span>Generator <span> /> </span></div>
+        </div>
+        <div class="card">
+            <img src="./img/rest.JPEG" alt="">
+            <div class="text"><span>
+                    < </span>Restaurant <span> /> </span></div>
+        </div>
+        <div class="card">
+            <img src="./img/rest1.JPEG" alt="">
+            <div class="text"><span>
+                    < </span>Food <span> /> </span></div>
+        </div>
```

Comment:
- [ ] As I mentioned above, kindly consider making the changes the same way I mentioned above briefly. the suggestions should be polite, and clear, and include the reason behind the required changes in a descriptive way.